core/services/wifi/main.py

```diff
@@ -208,5 +208,5 @@ def hotspot_state() -> Any:
     config = Config(app=app, loop=loop, host="0.0.0.0", port=9000, log_config=None)
     server = Server(config)
 
-    loop.create_task(wifi_manager.auto_reconnect(10.0))
+    loop.create_task(wifi_manager.auto_reconnect(60))
```

Comment:
Ouch, that's a lot of time. 
Can we adjust this value based if hotspot is running or not ?

Reply:
This is just a fallback, so we don't need to care much about it.